### ISSUE TITLE
JDK 8

### ISSUE THREAD

**Issue by micromachine** [Ericbla/check_jstat#8]
In JDK 8, jstat display Metaspace's statistics, but not PermGen's.

**Comment by rodmur** [comment]
yeah, this is kind of annoying, and now I'll be living in an environment that uses both jvm 1.7 and 1.8, whee...
